pages/book/statements.mdx

```diff
@@ -280,6 +280,15 @@ dump(sum); // 1000
   }
   ```
 
+  Trying to iterate over a member of a [Struct](/book/structs-and-messages#structs) also won't work:
```

Comment:
```suggestion
  Trying to iterate over a member of a [Struct](/book/structs-and-messages#structs) also won't work, because struct field access is a compound expression and not an identifier:
```